src/get-started/java-binding.md

```diff
@@ -1483,7 +1502,7 @@ For using the library just include the dependency in your `pom.xml`:
 - [Javadocs](https://exonum.com/doc/api/java-binding/0.9.0-rc2/index.html)
 - [Rust instruction](create-service.md)
 
-[abstractservice]: https://exonum.com/doc/api/java-binding/0.9.0-rc2/com/exonum/binding/core/service/AbstractService.html
+[abstract-service]: https://exonum.com/doc/api/java-binding/0.9.0-rc2/com/exonum/binding/core/service/AbstractService.html
```

Comment:
Do some links intentionally refer to `0.9.0-rc2` while others refer to `0.10.0`?

Reply:
Yes, I plan to replace all the 0.9.0-rc2 ones with 0.10.0 separately.